### PR TITLE
Final reorg of IWV section

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

- [x] ~1.6 Sections 6.2 and 6.3 (starting from Line 465): While in section 5.1, authors provided more thorough reviews and went over many different IWV-based COVID-19 vaccines, only two specific subunit vaccines, Nuvaxoid and Covifenz, were mentioned. Please consider including more subunit vaccines in Table 2 in the paragraphs and make them concise, as needed. This change may also impact Section 6.4, so please update Section 6.4 accordingly.~
- [x] 3.1 The manuscript is subdivided into sections and sub-sections with headings and subheadings, respectively. **Please ensure that the text under each subsection is reflecting the heading/subheading and that the information the reader intuitively seeks for under a specific heading is included in that heading.** 
- [x] If such information/data are not or not yet available, I would recommend mentioning the lack of such. For example(s), certain sections such as 6.4 the authors revisit clinical trials of Novavax and do not discuss safety. 
- [x] Please make the distinctions between trials and real-world data clear throughout the manuscript.
- [x] 3.2 As I understood, the authors generally aim to discuss the platform, dose/regimen, clinical trial efficacy and safety data in sections named Application to COVID-19 or ones with vaccine name (Covifenz, Nuvaxovid) and then RW effectiveness, safety and response/implications for VoC under sections named Real-World Safety and Efficacy. **I would first suggest using a uniform naming system for different platforms.** Some Real-World Safety and Efficacy sections go back to clinical trials and overview VoC, some are entirely dedicated to VoC. You could add VoC to the subheadings or, as with live attenuated vaccines section, have a separate subsection devoted to VoC, as large part of VoC-related text represents immune outcome data, not RWE per se. With, for example, CoronaVac, the real-world data from Chile is addressed in the more general section and not mentioned in the RW Safety and Efficacy section. As above, I would recommend clearly separating the source of data (trial vs real-world). Also, I suggest logically structuring the RW Safety and Efficacy section as follows: first - effectiveness data (or lack of such) shortly following vaccination, over time, as related to boosters, as pertaining to the VoC.
- [x] 3.3 The authors discuss efficacy/effectiveness, side effect profile, sometimes regulatory details for most of the vaccines. Please make sure each of these outcomes are addressed for each vaccine. For example, **RW safety and effectiveness information is lacking for IWV vaccines section, while all of them discuss VoC.** Similarly, safety and side effect details are not provided for Nuvaxovid, for example. As above, I recommend ensuring the data points given for each platform/vaccine are uniform; separating immune outcome data from the real-world effectiveness; mentioning the lack of data wherever relevant.
- [x] 3.6 When providing percent-efficacy or effectiveness, I recommend to consistently indicate the time period/interval over which the outcome is reported (already included in many cases).
- [x] 3.7 I would suggest using the same naming for vaccines after they (and their alternative names) are first mentioned. For example, Covilo and Sinopharm, CoronaVac and Sinovac are used interchangeably, and it can be confusing for some readers.
- [x] 3.9 Please make sure all relevant sentences are followed by a reference, where appropriate, e.g. (not limited to) lines 143, 191, 469.
- [x] 3.20 Lines 305-308 are identical to lines 324-327, please leave where more appropriate.
- [x] 3.22 Section 5.2 could generally benefit from a slight reorganization. Please see comments 1-3.
- [x] 3.25 Lines 349-364: Please, clearly separate the discussion of Sinovac and Covaxin.
- [x] ~3.30 Lines 409-411 are identical to lines 415-417, please leave where more appropriate.~

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/22.vaccines-trad.md [hunk 1]
```diff
@@ -127,36 +154,26 @@ A live attenuated recombinant strain of RSV previously investigated as a vaccine
...
-The updated vaccine was again tested in Syrian golden hamsters.
-No breakthrough infections were observed following vaccination with YF-S0\* and exposure to the index strain and the Alpha, Beta, Gamma, and Delta variants [@doi:10.1101/2021.11.12.468374].
-YF-S0\* also reduced the infectious viral load in the lungs of several VOCs (Alpha, Beta, Gamma, and Delta) relative to a sham comparison [@doi:10.1101/2021.11.12.468374], and the likelihood of the Delta variant spreading to unvaccinated co-housed hamsters was significantly reduced by YF-S0\* [@doi:10.1101/2021.11.12.468374].
-The updated vaccine was also associated with the increased production of nAbs against the Omicron variant compared to YF-S0 [@doi:10.1101/2021.11.12.468374].
-Therefore, while data from human studies are not available, preclinical results suggest that LAV vaccines likely confer some protection against VOC even when designed with the index strain.
-Modifications to the design may make this protection more robust as SARS-CoV-2 evolves.
-
 ### Inactivated Whole-Virus Vaccines
```
Comment (rando2) [inline content/22.vaccines-trad.md:177]: This is the section associated with this PR.

**Comment by rando2** [comment]
Hey @RLordan, one more after this! Please let me know if you catch that any of them are missing some info that the others have, I find it really challenging to make sure everything is balanced across sections!

**Comment by RLordan** [comment]
> Hey @RLordan, only one more PR to come after this! Please let me know if you catch that any of them are missing some info that the others have, I find it really challenging to make sure everything is balanced across sections!
> 
> The LAV section is showing up as new changes in the files changed tab, but I made a comment to show you where to start reading! The rest of the changes went through in #1189

Super thanks @rando2 will do asap

**Review by RLordan — APPROVED** [review]
Hey @rando2 minor suggestions. Thanks a mill.

**Comment by RLordan** [comment]
> Hey @RLordan, only one more PR to come after this! Please let me know if you catch that any of them are missing some info that the others have, I find it really challenging to make sure everything is balanced across sections!
> 
> The LAV section is showing up as new changes in the files changed tab, but I made a comment to show you where to start reading! The rest of the changes went through in #1189

I will try read it properly again too when all the PRs have gone through with reviewer comments in mind. It is hard to bring it all together :D

**Pull request merged**